test/MiniExec.t.sol

```diff
@@ -23,37 +37,55 @@ contract MiniExecTest is Test, MiniExecScript {
         account = MiniExecProxy(payable(factory.createAccount(remoteOwner, remoteNetworkId)));
     }
 
-    // @OF: more extensive testing please
-    // @OF: for example use the following contract, and test calls that will succeed and revert (and check that referenceValue changes)
-    /*
-    contract MockContract { 
-        address immutable referenceAddr;
-        uint256 public referenceValue;
-
-        constructor(address a) {
-            referenceAddr = a;
-        }
-
-        function someFunction(address a, uint256 u) {
-            assert(a == referenceAddr);
-            referenceValue = u;
-        }
+    function testCallsCorrectly() external {
```

Comment:
the goal with `someFunction` was that you'd test something that passes values and changes the storage ;)

Reply:
can do that, but this tests that calldata is passed correctly, no difference between calldata being just a selector or selector and encoded params